core/src/main/res/values/styles.xml

```diff
@@ -1,11 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
-<resources xmlns:tools="http://schemas.android.com/tools" xmlns:android="http://schemas.android.com/apk/res/android">
+<resources xmlns:android="http://schemas.android.com/apk/res/android" xmlns:tools="http://schemas.android.com/tools">
 
     <style name="Theme.AntennaPod.Light" parent="Theme.Base.AntennaPod.Light">
         <!-- Room for API dependent attributes -->
     </style>
 
     <style name="Theme.Base.AntennaPod.Light" parent="Theme.MaterialComponents.Light">
+        <item name="filter_dialog_button_background">
+            @drawable/filter_dialog_button_background_light
+        </item>
```

Comment:
Please also move this to the bottom. Why did you add line breaks here? The other theme definitions do not use line breaks.

Reply:
how did that happen?